### PR TITLE
feat(records): deco records

### PR DESCRIPTION
**Pull request by igorbrasileiro**

### CODE REVIEW

File: turso/mod.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+export interface Props extends StorageConfig {
+}
+
+/**
+ * @title Turso
+ * @description App for using turso https://turso.tech/
+ * @category Tool
+ * @logo todo
+ */
+export default function Turso(
```
Comment (lucis) [inline turso/mod.ts:41]: Why `Turso` and not `Sqlite`?

**Review by lucis — APPROVED** [review]

File: turso/mod.ts (hunk 1 above)
Comment (lucis) [reply turso/mod.ts:41]: And why the two apps and not only one?

File: records/scripts/pullProd.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+async function run() {
+  if (!token || !sitename) {
+    console.error(
+      "DATABASE_AUTH_TOKEN and DECO_SITE_NAME environment variables must be set in order to pull the production database.",
+    );
+    throw error;
+  }
+
+  const productionDumpUrl = `https://${sitename}-decocx.turso.io/dump`;
```
Comment (lucis) [inline records/scripts/pullProd.ts:14]: [Shouldn't you get this from the app's config?https://www.getlago.com/](https://www.getlago.com/)

File: records/scripts/pullProd.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+  if (!response.ok) {
+    console.error("Could not connect to production database. Make sure your environment variables DATABASE_AUTH_TOKEN and DECO_SITE_NAME are up to date.");
+    console.log("Tip: You can copy a fresh DATABASE_AUTH_TOKEN variable from the Deco.cx admin.");
+    throw error;
+  }
+
+  const dumpQuery = await response.text();
+
+  const command = new Deno.Command("sqlite3", {
```
Comment (lucis) [inline records/scripts/pullProd.ts:37]: Is this a global exec?

**Review by lucis — APPROVED** [review]

File: records/scripts/pullProd.ts (hunk 2 above)
Comment (igorbrasileiro) [reply records/scripts/pullProd.ts:14]: This runs outside the deco flow. Do you to suggest that should be inside the deco flow?

File: records/scripts/pullProd.ts (hunk 3 above)
Comment (igorbrasileiro) [reply records/scripts/pullProd.ts:37]: @viktormarinho can you explain this?

**Pull request merged**